packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/renditions/RenditionIntegrationTests.java

```diff
@@ -68,15 +71,23 @@ protected void checkRendition(String fileName, String nodeId, String renditionId
      */
     protected RestNodeModel uploadFile(String sourceFile) throws Exception
     {
-        FolderModel folder = FolderModel.getRandomFolderModel();
-        folder = dataContent.usingUser(user).usingSite(site).createFolder(folder);
+        FolderModel folder = Awaitility
+            .await()
+            .atMost(Duration.ofSeconds(30))
+            .pollInterval(Duration.ofSeconds(6))
```

Comment:
Can we use a smaller `pollInterval`, e.g.: `Duration.ONE_SECOND`?

Reply:
Yes. Done.